plugins/source/cloudflare/docs/index.md

```diff
@@ -1,54 +0,0 @@
-## Cloudflare Plugin
-
-The CloudQuery Cloudflare provider pulls configuration out of Cloudflare resources, normalizes them and stores them in PostgreSQL database.
-
-### Install
-
-```bash
-cloudquery init cloudflare
-```
-
-### Configuration
-
-The following configuration section can be automatically generated by `cloudquery init cloudflare`:
-
-```yaml
-// Use can use either the API token or the API key
-// API token is preferred
-
-// API token to access Cloudflare resources, also can be set with the CLOUDFLARE_API_TOKEN environment variable
-api_token: "<YOUR_API_TOKEN_HERE>"
-// API key to access Cloudflare resources, also can be set with the CLOUDFLARE_API_KEY environment variable
-api_key: "<YOUR_API_KEY_HERE>"
-// API email to access Cloudflare resources, also can be set with the CLOUDFLARE_API_EMAIL environment variable
-api_email: "<YOUR_API_EMAIL_HERE>"
-
-// List of accounts to target, if empty, all accounts will be targeted
-//accounts:
-// - "<YOUR_ACCOUNT_ID>"
-
-// List of accounts to target, if empty, all available zones will be targeted
-//zones:
```

Comment:
This example should also be changed to use `yaml`-style comments.

Reply:
Done.